src/query/storage/m3/storage.go

```diff
@@ -62,15 +61,15 @@ var (
 
 type m3storage struct {
 	clusters Clusters
-	opts     m3db.Options
+	opts     Options
```

Comment:
this was entire point of the change. now in the future we can types from `storage/m3`, like `ClusterNamespaces` to the `Options`.